smif/model/sos_model.py

```diff
@@ -162,8 +162,10 @@ def simulate(self, timestep, data=None):
                 sim_data[input_.name] = param_data_converted
 
             # Pass in parameters to contained model
+            default_data = model.parameters.defaults
             if data and model.name in data:
-                sim_data.update(data[model.name])
+                param_data = dict(default_data, **data[model.name])
+                sim_data.update(param_data)
```

Comment:
Should there be an `else:` clause here that passes in the defaults - `sim_data.update(default_data)`?